### PR TITLE
Remove "Nothing to do" log entry

### PR DESCRIPTION
**Pull request by reggieb**
https://github.com/rails/webpacker/issues/2392

Removes the "Nothing to do" log entry from the `Webpacker::Compiler#compiler` method.

Nothing to do should include logging!

### CODE REVIEW

File: lib/webpacker/compiler.rb [hunk 1]
```diff
@@ -27,7 +27,6 @@ def compile
         record_compilation_digest
       end
     else
```
Comment (jakeNiemiec) [inline lib/webpacker/compiler.rb]:
```suggestion
     else
       logger.debug "Everything's up-to-date. Nothing to do"
```

**Comment by rossta** [comment]
@reggieb This feels like a good change to me. Want to accept the requested change so we can try to get this through?

**Comment by reggieb** [comment]
@rossta I've reinstated the logger call with the level set to debug

**Comment by nick123pig** [comment]
This would be very helpful to merge in when able!

**Comment by david-mears-dfe** [comment]
@jakeNiemiec Tagging to bring this to your attention. You'd be very popular if you found time to re-review and merge!

**Comment by reggieb** [comment]
Is there a reason why this has been closed without merging?

**Comment by gauravtiwari** [comment]
Sorry @reggieb 

Was just trying to cleanup PRS and issues so we can look things afresh post 6.0 release.

**Comment by reggieb** [comment]
@gauravtiwari No problem.

I'm really please this has finally been merged. Thank you.

**Pull request merged**